po/zh.po

```diff
@@ -1614,72 +1579,66 @@ msgid "Start cmdline with 9"
 msgstr ""
 
 #: src/keymap.cpp:438
-#, fuzzy
 msgid "Move to the previous entry"
-msgstr "转到前一篇未读的种子"
+msgstr "移动到上一个条目(entry)"
 
 #: src/keymap.cpp:439
-#, fuzzy
 msgid "Move to the next entry"
-msgstr "转到下一篇未读文章"
+msgstr "移动到下一个条目(entry)"
 
 #: src/keymap.cpp:444
-#, fuzzy
 msgid "Move to the previous page"
-msgstr "返回到前一个对话框/退出"
+msgstr "翻到上一页"
 
 #: src/keymap.cpp:451
-#, fuzzy
 msgid "Move to the next page"
-msgstr "转到下一篇未读文章"
+msgstr "翻到下一页"
 
 #: src/keymap.cpp:454
 msgid "Move half page up"
-msgstr ""
+msgstr "向上翻半页"
 
 #: src/keymap.cpp:455
 msgid "Move half page down"
-msgstr ""
+msgstr "向下翻半页"
 
 #: src/keymap.cpp:461
-#, fuzzy
 msgid "Move to the start of page/list"
-msgstr "转到下一篇未读文章"
+msgstr "移动到这一页/这个列表的开头"
 
 #: src/keymap.cpp:468
-#, fuzzy
 msgid "Move to the end of page/list"
-msgstr "转到下一篇未读文章"
+msgstr "移动到这一页/这个列表的结尾"
 
 #: src/keymap.cpp:711
 #, c-format
 msgid "`%s' is not a valid context"
-msgstr ""
+msgstr "`%s' 不是一个有效的上下文环境(context)"
 
 #: src/keymap.cpp:716
 #, c-format
 msgid "`%s' is not a valid key command"
-msgstr ""
+msgstr "`%s' 不是一个有效的键盘指令"
 
 #: src/keymap.cpp:764
-#, fuzzy, c-format
+#, c-format
 msgid "failed to parse operation sequence for %s"
-msgstr "当调用`%s'的时候出错: %s"
+msgstr "解析操作序列失败：%s"
```

Comment:
> it could be just a relic from the previous fuzzy version

Yes, it is a relic and also an acceptable translation. I think `%s` refers to the text of the `operation sequence`, so it's also alright to place it just after the `操作序列` which means `operation sequence`. Updated in the latest commit.